File: SDKs/amazonaws.com/sagemaker/2017-07-24/go/pkg/models/shared/frameworkenum.go
package shared

type FrameworkEnum string

const (
	FrameworkEnumTensorflow FrameworkEnum = "TENSORFLOW"
	FrameworkEnumKeras      FrameworkEnum = "KERAS"
	FrameworkEnumMxnet      FrameworkEnum = "MXNET"
	FrameworkEnumOnnx       FrameworkEnum = "ONNX"
	FrameworkEnumPytorch    FrameworkEnum = "PYTORCH"
	FrameworkEnumXgboost    FrameworkEnum = "XGBOOST"
	FrameworkEnumTflite     FrameworkEnum = "TFLITE"
	FrameworkEnumDarknet    FrameworkEnum = "DARKNET"
	FrameworkEnumSklearn    FrameworkEnum = "SKLEARN"
)

File: SDKs/amazonaws.com/sagemaker/2017-07-24/python/src/sdk/models/shared/framework_enum.py
from dataclasses import dataclass, field
from enum import Enum

class FrameworkEnum(str, Enum):
    TENSORFLOW = "TENSORFLOW"
    KERAS = "KERAS"
    MXNET = "MXNET"
    ONNX = "ONNX"
    PYTORCH = "PYTORCH"
    XGBOOST = "XGBOOST"
    TFLITE = "TFLITE"
    DARKNET = "DARKNET"
    SKLEARN = "SKLEARN"

